addon/PageScraper.js

```diff
@@ -73,6 +73,7 @@ PageScraper.prototype = {
           };
           resolve([imageWithSize]);
         });
+        image.addEventListener("error", () => reject());
```

Comment:
Do we want to `reject` here? We seem to `resolve` with a default of 500x500 above. Also, if we do `reject`, it will `reportError` https://github.com/sarracini/activity-stream/blob/ff85a9960a53704f3c10a700ab04852b1ff0da8e/addon/PageScraper.js#L47 and is that desired?